yoti/Readme.md

```diff
@@ -0,0 +1,156 @@
+# Yoti
+
+Welcome to the Yoti Drupal SDK. This repo contains the tools you need to
+quickly integrate your Drupal back-end with Yoti, so that your users can
+share their identity details with your application in a secure and trusted
+way.
+
+## Table of Contents
+
+1) [An Architectural view](#an-architectural-view) -
+High level overview of integration
+
+2) [References](#references) -
+Guides before you start
+
+3) [Installing the SDK](#installing-the-sdk) -
+How to install our SDK
+
+4) [Plugin Setup](#plugin-setup)-
+How to set up the plugin in Drupal
+
+5) [Linking existing accounts to use Yoti authentication
+](#linking-existing-accounts-to-use-yoti-authentication)
+
+6) [API Coverage](#api-coverage) -
+Attributes defined
+
+7) [Support](#support) -
+Please feel free to reach out
+
+## An Architectural view
+
+Before you start your integration, here is a bit of background on how the
+integration works. To integrate your application with Yoti, your back-end
+must expose a GET endpoint that Yoti will use to forward tokens.
+
+The endpoint can be configured in the Yoti Dashboard when you create/update
+your application. For more information on how to create an application please
+check our [developer
+page](https://www.yoti.com/developers/documentation/#login-button-setup).
```

Comment:
URL change I mentioned in https://github.com/getyoti/yoti-drupal/pull/39 👍

Reply:
Fixed in fd1ea74